packages/pwa-buildpack/lib/queries/index.js

```diff
@@ -1,9 +1,22 @@
-// Execute this first so we can import .graphql files directly.
-const requireGraphQL = require('require-graphql-file');
+const fs = require('fs');
+const path = require('path');
+
+/**
+ * Normal `require` doesn't know what to do with .graphql files, so this helper function
+ * simply imports their contents as a string.
+ * @see https://github.com/apollographql/apollo-server/issues/1175#issuecomment-397257339.
+ *
+ * @param   {String} filepath - A relative path to a .graphql file to read.
+ * @returns {String} - The contents of the file as a string.
+ */
+const requireGraphQL = filePath => {
```

Comment:
Can we call this `requireFile` instead of `requireGraphQL`? Also, move it to its own file and have some tests around it. This can be useful even for non GraphQL files.

Reply:
Honestly I considered that but couldn't think of another use case - do you have one in mind?

I left it here for now thinking that if we want to extract / generalize it later we can.

Reply:
Optimize later, IMO :)